# goal defines
int8 is_ready
---
# result defines
bool success
---
# feedback
int8 state

# state defines
int8 INITIALIZING=0
int8 PLANNING=1
int8 EXECUTING=2
